config/requests.yml

```diff
@@ -19,7 +19,7 @@ defaults: &defaults
 development:
   <<: *defaults
   voyager_ub_id: 1@DB
-  bibdata_base: <%= ENV['BIBDATA_BASE'] || "https://bibdata-staging.princeton.edu" %>
+  bibdata_base: <%= ENV['BIBDATA_BASE'] || "https://bibdata.princeton.edu" %>
```

Comment:
Was this intended to be committed?  It is what I use locally, so I have not issue with it changing...

Reply:
It was unintentional but I can keep the change since we both use it locally.